src/bindings/jna/README.md

```diff
@@ -115,17 +115,12 @@ Most likely, you have to include this dependency as well:
 
 ### Using Elektra Plugins
 
-The `NativeElektraPlugin` can be used to load a native Elektra plugin which is written in C/C++.
-You can load a Plugin like the following:
+The `PluginLoader` can be used to load a native Elektra plugin which is written in C/C++.
+You can load a native Elektra Plugin like the following:
 
 ```java
-Key parentKey = Key.create("user/tests/javabinding");
-NativeElektraPlugin yourPlugin = null;
-		try {
-			yourPlugin = new NativeElektraPlugin("error", parentKey);
-		} catch (InstallationException e) {
-			// Most likely the desired plugin is not installed or you had a typo in the pluginName
-		}
+PluginLoader pluginLoader = new PluginLoader();
+Plugin errorPlugin = pluginLoader.loadElektraPlugin("error");
```

Comment:
If they are checked exceptions it is not needed to mention as the compiler will check for this anyway. Otherwise, you should mention in text that this error might occur.

Reply:
It is a checked exception so it is fine